NumberInput/examples/NumberInput.sol

```diff
@@ -0,0 +1,24 @@
+pragma ton-solidity ^0.35.0;
+
+interface INumberInput {
+
+	function get(uint32 answerId, string prompt, int256 min, int256 max) external returns (int256 value);
+
+}
+
+library NumberInput {
+
+	uint256 constant ID = 0x05606e612a821f3ecf72621a28304c5ae5fcf7681637bf9aaf74954a190175b5;
```

Comment:
Please, fix it to `c5a9558b2664aed7dc3e6123436d544f13ffe69ab0e259412f48c6d1c8588401`

Reply:
> Please, fix it to `c5a9558b2664aed7dc3e6123436d544f13ffe69ab0e259412f48c6d1c8588401`

```suggestion
	uint256 constant ID = 0xc5a9558b2664aed7dc3e6123436d544f13ffe69ab0e259412f48c6d1c8588401;
```